atomic_dbus.py

```diff
@@ -44,16 +67,125 @@ def version(self, images, recurse=False):
+        storage = Storage()
+        args = self.Args()
+        args.graph = graph
+        args.import_location = import_location
+        storage.set_args(args)
+        storage.Import()
+
+    """
+    The storage_export method exports all containers and their associated contents into a filesystem directory.
+    """
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='ssb', out_signature='')
+    def storage_export(self, graph="/var/lib/docker", export_location="/var/lib/atomic/migrate", force = False):
+        storage = Storage()
+        args = self.Args()
+        args.graph = graph
+        args.export_location = export_location
+        args.force = force
+        storage.set_args(args)
+        storage.Export()
+
+    """
+    The storage_modify method modifies the default storage setup.
+    """
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='asv', out_signature='')
+    def storage_modify(self, devices=[], driver = None):
+        storage = Storage()
+        args = self.Args()
+        args.devices = devices
+        args.driver = driver
+        storage.set_args(args)
+        storage.modify()
+
+    """
+    The diff method shows differences between two container images, file diff or RPMS.
+    """
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='ssb',
+                         out_signature='a{sas}')
+    def diff(self,first,second, rpms=False):
+        diff = Diff()
+        args = self.Args()
+        args.compares.append(first)
+        args.compares.append(second)
+        args.json = False
+        args.no_files = False
+        args.names_only = False
+        args.rpms = rpms
+        args.verbose = True
+        diff.set_args(args)
+        return diff.diff()
+
+    """
+    The top method shows top-like stats about processes running inside containers.
+    """
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='asiasi',
+                         out_signature='')
+    def top(self, containers = [], d=1, optional=[], n=1):
+        top = Top()
+        args = self.Args()
+        args.containers = containers
+        args.d = d
+        args.optional = optional
+        args.n = n
+        top.set_args(args)
+        top.atomic_top()
+
+    """
+    The scan method scans an image or container for CVEs.
+    """
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='asasbbasbasb',
+                         out_signature='')
+    def scan(self, scan_targets=[], scanners=None, _list=False, verbose=False, rootfs=[], all=False, images=False, containers=False ):
+        scan = Scan()
+        args = self.Args()
+        args.scan_targets = scan_targets
+        args.scanners = scanners
+        args.list = _list
```

Comment:
Change _list to _list_scanners, but this should be a separate function.  scan_list().  Not used for scanning.